### PR TITLE
Bug 1177254 - Add visibleaudiochannelchanged and audiochannelchangedasactive events to instead of the MozChromeEvent events

### PR DESCRIPTION
**Pull request by evanxd**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cf1f490d450c662adca1fe16a04c26165b70f236

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=58c5cd0a613c75e8abe4924bae3dc42844735844

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=01bc84cf57a8a3cc5d43c6360fb1cd9383b9e862

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=31850a75734bab557d6eda1340485eea070c2390

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=df27f5267e18a507240174a371e31c0f137696d8

File: apps/system/js/sound_manager.js [hunk 1]
```diff
@@ -351,16 +353,16 @@
...
       case 'mozChromeEvent':
         switch (e.detail.type) {
           case 'bluetooth-volumeset':
             this.changeVolume(e.detail.value - this.currentVolume.bt_sco,
                               'bt_sco');
             break;
-          case 'audio-channel-changed':
-            this.setAudioChannel(e.detail.channel);
-            this.ceAccumulator();
-            break;
```
Comment (dominickuo) [inline apps/system/js/sound_manager.js]: @evanxd Before the new audio channel api actually landed in gecko, we should keep this `audio-channel-changed` case in gaia and make sure it still works on master branch with current gecko.

File: apps/system/js/visibility_manager.js [hunk 2]
```diff
@@ -121,24 +121,22 @@
...
-
-          if (this._normalAudioChannelActive &&
-              evt.detail.channel !== 'normal' && Service.query('locked')) {
-            this._deviceLockedTimer = setTimeout(function setVisibility() {
-              if (window.Service.query('locked')) {
-                this.publish('hidewindow',
-                  { screenshoting: false, type: evt.type });
-              }
-            }.bind(this), 3000);
-          }
```
Comment (dominickuo) [inline apps/system/js/visibility_manager.js]: Same here, let's keep the `mozChromeEvent` case for now, to make sure it still works on the master branch with latest gecko.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2eecd961cce9a03f190eb10f023fe08f9b860be0

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=77940d4c6fd8767547df5e66d7a77f58b34c15d1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9b0ff8f0a4db0fcc40d3252edfe3738a78dbef3e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=02de426c583b3d8d31988a918e0601ee4d4db978

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=07f5918578c8fb3ef1c7fde20aae9012b972ec01

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4de39935842f84610dc599398b5ca8a356b5fac6

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=401b4921079fed3c372d4a9415789517c755f15a

**Pull request merged**